build.gradle

```diff
@@ -1111,8 +1113,7 @@ project(':core') {
   }
 
   tasks.create(name: "copyDependantLibs", type: Copy) {
-    from (configurations.testRuntimeClasspath) {
```

Comment:
`copyDependantLibs` should not use `testRuntimeClasspath` since it causes circular dependencies. `jar` -> `copyDependantLibs` -> `test jar` -> `jar` -> `copyDependantLibs` ...

The purpose of this copy is to collect `reload4j*jar`, so we can get it from compile path instead